doc/nrf/ug_tfm.rst

```diff
@@ -48,18 +52,33 @@ When building for ``nrf9160dk_nrf9160ns``, UART1 must be disabled in the non-sec
 Otherwise, the non-secure application will fail to run.
 The recommended way to do this is to copy the .overlay file from the :ref:`tfm_hello_world` sample.
 
+Enabling secure services
+========================
+
+When using the :ref:`nrfxlib:nrf_security`, if :option:`CONFIG_BUILD_WITH_TFM` is enabled together with :option:`CONFIG_NORDIC_SECURITY_BACKEND`, the TF-M secure image will enable the use of the hardware acceleration of Arm CryptoCell.
+In such case, the Kconfig configurations in the Nordic Security Backend control the features enabled through TF-M.
+
+You can configure what crypto modules to include in TF-M by using the ``TFM_CRYPTO_`` Kconfig options found in file :file:`zephyr/modules/trusted-firmware-m/Kconfig.tfm.crypto_modules`.
+
+TF-M utilizes :ref:`hardware unique keys <lib_hw_unique_key>` when the PSA Crypto key derivation APIs are used.
```

Comment:
```suggestion
TF-M utilizes :ref:`hardware unique keys <lib_hw_unique_key>` when the PSA Crypto key derivation APIs are used, and psa_key_derivation_setup is called with the alg TFM_CRYPTO_ALG_HUK_DERIVATION
```

Reply:
Done, I just spelled out algorithm instead of alg.